src/browser/api/system.js

```diff
@@ -492,6 +496,12 @@ exports.System = {
     },
 
     showChromeNotificationCenter: function() {},
+    startCrashReporter: function(identity, options) {
```

Comment:
mmmmmmm i will if at least one more person wants it... id rather them be consistent and i dont want to change them all